include/stronk/stronk.h

```diff
@@ -143,6 +191,30 @@ struct can_multiply
     }
 };
 
+template<typename T>
+struct can_multiply_with
+{
+    template<typename StronkT>
+    struct skill
+    {
+        constexpr friend auto operator*=(StronkT& lhs, const T& rhs) noexcept -> StronkT
+        {
+            lhs.template unwrap<StronkT>() *= rhs;
+            return lhs;
+        }
+
+        constexpr friend auto operator*(const StronkT& lhs, const T& rhs) noexcept -> StronkT
+        {
+            return StronkT {lhs.template unwrap<StronkT>() * rhs};
+        }
+
+        constexpr friend auto operator*(const T& lhs, const StronkT& rhs) noexcept -> StronkT
```

Comment:
Do you need both of these (T and StronkT swapped)? I thought that was what the friend operator handled compared to a member operator.

Reply:
Ill test but I think it actually requires it - i dont know if it can assume that * is commutative.

Reply:
it had to be there